src/pybind/mgr/dashboard/frontend/src/app/shared/components/alert-panel/alert-panel.component.html

```diff
@@ -1,6 +1,6 @@
 <ngb-alert type="{{ bootstrapClass }}"
            [dismissible]="dismissible"
-           (close)="onClose()">
```

Comment:
Is this a change in the ngb-alert API?

Reply:
After bootstrap 8.0.0 I saw this behavior is changed and close was no longer working. https://ng-bootstrap.github.io/#/components/alert/api

Reply:
Ah, they changed the output `close` with `closed`, and now `close` is a method to close the alert... Thanks!